client/src/showcase-api/index.d.ts

```diff
@@ -0,0 +1,3 @@
+export * from './runtime'
```

Comment:
No need to review showcase-api folder, it's a copy of credential-showcase-openapi/dist